### ISSUE TITLE
features

### ISSUE THREAD

**Issue by mMdooD** [aDarkDev/NotPixel#97]
U can add do tasks and you can make it paint the shape that bot suggested because it gives 3 times the points and tnxxxx <3

**Comment by aDarkDev** [comment]
Yes, I can, but right now I am going through some tough days. I will definitely get to continue this project.
